WireAPI/Tests/WireAPITests/APIs/FeatureConfigsAPI/FeatureConfigsAPITests.swift

```diff
@@ -437,11 +425,11 @@ extension FeatureConfigsAPITests {
 
 }
 
-private extension APIVersion {
-
-    func buildAPI(client: any HTTPClient) -> any FeatureConfigsAPI {
-        let builder = FeatureConfigsAPIBuilder(httpClient: client)
-        return builder.makeAPI(for: self)
-    }
-
-}
+// private extension APIVersion {
+//
+//    func buildAPI(client: any APIServiceProtocol) -> any FeatureConfigsAPI {
+//        let builder = FeatureConfigsAPIBuilder(apiService: apiService)
+//        return builder.makeAPI(for: self)
+//    }
+//
+// }
```

Comment:
I guess this should be deleted

Reply:
done in https://github.com/wireapp/wire-ios/pull/2249/commits/a4c9b508d43ee8d6c87dfdf0ad4952daa8f6ec78